README.md

```diff
@@ -84,6 +84,23 @@ compared as plain text. This is useful for a convenient
 low-storage way of making sure that the return value doesn't
 change for selected test cases.
 
+## Updating References within Package Tests
+
+Reference tests are typically used within a package's `test/runtests.jl`
+test suite. These tests are easy to run via `pkg> test` but
+the child process used within `pkg> test` is non-interactive, so the
+update prompt will not show if there are mismatches.
+
+To update references within a package test suite, there are two options:
+
+1. Set the environment variable `JULIA_REFERENCETESTS_UPDATE` to `"true"`
+   and run `pkg> test`, which will force update any non-matches. You can then
+   check changes to any git-tracked reference images before commit.
```

Comment:
Before this; we usually remove the entire `test/references` folder and track the git differences :)

But this is seems a better approach.